sdk/communication/Azure.Communication.Chat/README.md

```diff
@@ -65,7 +65,12 @@ Once you initialized a `ChatClient` class, you can do the following chat operati
 
 ### Create a thread
```

Comment:
You'll also need to update the Changelog file before you merge and release this version

Reply:
Yes you are right. I  will create a separate PR for that.

Reply:
I see that Lu did it in the same PR